cicecore/cicedynB/general/ice_init.F90

```diff
@@ -422,7 +427,8 @@ subroutine input_data
       reltol_fgmres = 1e-2_dbl_kind ! fgmres stopping criterion: reltol_fgmres*res(k)
       reltol_pgmres = 1e-6_dbl_kind ! pgmres stopping criterion: reltol_pgmres*res(k)
       algo_nonlin = 'picard'        ! nonlinear algorithm: 'picard' (Picard iteration), 'anderson' (Anderson acceleration)
-      fpfunc_andacc = 1        ! fixed point function for Anderson acceleration: 1: g(x) = FMGRES(A(x),b(x)), 2: g(x) = x - A(x)x + b(x)
+      fpfunc_andacc = 1        ! fixed point function for Anderson acceleration: 1: 
+                               ! g(x) = FMGRES(A(x),b(x)), 2: g(x) = x - A(x)x + b(x)
```

Comment:
the `1: ` should be on the second line  here too

Reply:
fixed